tests/test_mappings/test_imports/test_modules/test_base.py

```
@@ -413,3 +414,55 @@ def test_resolve_expense_attribute_errors(db):
+
+    project = DestinationAttribute.objects.filter(workspace_id=workspace_id, attribute_type='PROJECT', value='Mobile App Redesign').first()
+    assert project.active == True
+
+    mocker.patch(
+            'sageintacctsdk.apis.Projects.get_all_generator',
+            return_value=destination_attributes_data['create_new_auto_create_projects_destination_attributes_inactive_active']
+        )
+    
+    sage_intacct_connection.sync_projects()
+    project = DestinationAttribute.objects.filter(workspace_id=workspace_id, attribute_type='PROJECT', value='Mobile App Redesign').first()
+    assert project.active == False
```

Ruff

Avoid equality comparisons to `False`; use `if not project.active:` for false checks

Replace with `not project.active`

(E712)